db/migrate/20250608013722_create_active_operator_operations.rb

```diff
@@ -0,0 +1,15 @@
+class CreateActiveOperatorOperations < ActiveRecord::Migration[8.0]
+  def change
+    create_table :active_operator_operations do |t|
+      t.references :record, polymorphic: true, null: false
+      t.string :name
+      t.integer :version
+      t.json :response
```

Comment:
Any reason we aren't marking these as non-null? I also wonder if the response column should default to being an empty hash, though I can't figure out if that's better for querying?

```suggestion
      t.string :name, null: false
      t.integer :version, null: false
      t.json :response, null: false, default: "{}"
```

Reply:
Oh, for sure! This was just an oversight as I was in sketch mode. Thanks!